data/feb_fki_opening.csv

```diff
@@ -0,0 +1,3 @@
+fsn,warehouse,unmoved_quantity,invoice_ref,po_ref,unit_price_without_tax,unit_tax_amount,tax_rate,price_type,component_id,currency,grn_ref,mm_valuations_valuation_ref_id,consignment_ref,dn_ref,irn_ref,mm_valuation_parent_ref_id,mm_valuation_transaction_date,order_ref,rasl_event_id,rasl_event_type,ro_ref,seller_id,shipment_ref,sku,source_event_date,srn_ref,stock_type,variance_ref,vs_ref,entity_id,tax_type,actual_unit_price_without_tax,actual_unit_tax_amount,actual_unit_price_without_tax,igst_tax_rate,igst_unit_tax_amount,igst_actual_unit_tax_amount,sgst_tax_rate,sgst_unit_tax_amount,sgst_actual_unit_tax_amount,cgst_tax_rate,cgst_unit_tax_amount,cgst_actual_unit_tax_amount,cess_tax_rate,cess_unit_tax_amount,cess_actual_unit_tax_amount,mrp,hsn,return_type,return_ref,mm_evnt_event,mm_evnt_quantity,material_group_id
```

Comment:
Read this file from the FTP directory where you  saved the file after fetching from Bigfoot.